packages/api/lib/errors.js

```diff
@@ -11,11 +11,14 @@ const {
 // https://www.postgresql.org/docs/10/errcodes-appendix.html
 const isPostgresValidationError = (error) => ['22', '23'].includes((error.code || '').substring(0, 2));
 
+const isDynamoValidationException = (error) => error.name === 'ValidationException';
+
 const isBadRequestError = (error) =>
   error.name === 'SchemaValidationError'
   || error.name === 'ValidationException'
   || error instanceof ValidationError
-  || isPostgresValidationError(error);
+  || isPostgresValidationError(error)
+  || isDynamoValidationException(error);
```

Comment:
This seems fine, but we might want a TODO to remove it when the P3 work is done?  🤔